Airports/EGKK/Positions.txt

```diff
@@ -4,9 +4,13 @@ EGKK-S_APP:Gatwick Director:126.825:KKS:S:EGKK-S:APP:-:-:3750:3761:N051.08.53.00
 EGKK-S_APP:Gatwick Director:126.825:KKS:S:EGKK+S:APP:-:-:3750:3761:N051.08.53.000:W000.11.25.000::
 EGKK-S_APP:Gatwick Director:126.825:KKS:S:EGKK~S:APP:-:-:3750:3761:N051.08.53.000:W000.11.25.000::
 EGKK_F_APP:Gatwick Director:118.950:KKF:K:EGKK:APP:-:-:3750:3761:N051.08.53.000:W000.11.25.000::
-EGKK_TWR:Gatwick Tower:124.225:KKT:K:EGKK:TWR:-:-:3750:3761:N051.08.53.000:W000.11.25.000::
-EGKK_GND:Gatwick Ground:121.800:KKG:K:EGKK:GND:-:-:3750:3761:N051.08.53.000:W000.11.25.000::
-EGKK_P_GND:Gatwick Planner:134.225:KKP:K:EGKK:GND:-:-:3750:3761:N051.08.53.000:W000.11.25.000::
-EGKK_P_GND:Gatwick Planner:134.225:KKP:K:EGKK:DEL:-:-:3750:3761:N051.08.53.000:W000.11.25.000::
-EGKK_DEL:Gatwick Delivery:121.950:KKD:K:EGKK:DEL:-:-:3750:3761:N051.08.53.000:W000.11.25.000::
+EGKK_TWR:Gatwick Tower:124.230:KKT:K:EGKK:TWR:-:-:3750:3761:N051.08.53.000:W000.11.25.000::
+EGKK_TWR:Gatwick Tower:124.225:ERROR:K:EGKK:TWR:-:-:3750:3761:N051.08.53.000:W000.11.25.000::
+EGKK_GND:Gatwick Ground:121.805:KKG:K:EGKK:GND:-:-:3750:3761:N051.08.53.000:W000.11.25.000::
+EGKK_GND:Gatwick Ground:121.800:ERROR:K:EGKK:GND:-:-:3750:3761:N051.08.53.000:W000.11.25.000::
+EGKK_P_GND:Gatwick Planner:134.230:KKP:K:EGKK:GND:-:-:3750:3761:N051.08.53.000:W000.11.25.000::
+EGKK_P_GND:Gatwick Planner:134.225:ERROR:K:EGKK:GND:-:-:3750:3761:N051.08.53.000:W000.11.25.000::
+EGKK_P_GND:Gatwick Planner:134.225:ERROR:K:EGKK:DEL:-:-:3750:3761:N051.08.53.000:W000.11.25.000::
```

Comment:
```suggestion
EGKK_P_GND:Gatwick Planner:134.230:KKP:K:EGKK:DEL:-:-:3750:3761:N051.08.53.000:W000.11.25.000::
EGKK_P_GND:Gatwick Planner:134.225:ERROR:K:EGKK:DEL:-:-:3750:3761:N051.08.53.000:W000.11.25.000::
```
Just to maintain consistency with what we had before functionality wise. I don't think it would ever be used, but don't wanna break something inadvertantly.

If others think it isn't needed anymore we could just remove this DEL type position definition perhaps?

Reply:
I added Gatwick Planner with **both** tags (GND and DEL). @PLM1995 on Discord said that's indeed functionality wise so that station can be opened with either GND or DEL suffix.